storyscript/exceptions/WarningDeprecation.py

```diff
@@ -0,0 +1,129 @@
+# -*- coding: utf-8 -*-
+import click
+
+from .DeprecationCodes import DeprecationCodes
+from .Diagnostics import Diagnostics
+from .StoryError import StoryError
+
+
+class WarningDeprecation(Diagnostics):
```

Comment:
Deprecation != Warning (many compilers have both concepts, so we need to be a bit careful with the wording here).

```suggestion
class DeprecationMessage(Diagnostics):
```